CHANGELOG.md

```diff
@@ -70,6 +70,43 @@ $govuk-suppressed-warnings: (
 
 This was added in [#2911 Add warning suppression functionality](https://github.com/alphagov/govuk-frontend/pull/2911)
 
+#### Configure components in JavaScript
+
+JavaScript components can get the same configuration options in 2 ways - through data attributes, as before, and now when creating an instance. These components are:
+
+- the `Button` component, for its `preventDoubleClick` option (matching `data-prevent-double-click`)
+- the `CharacterCount` component, for its `maxlength`,`maxwords` and `threshold` options (matching `data-maxlength`, `data-maxwords` and `data-threshold`, respectively)
+- the `ErrorSummary` component, for its `disableAutoFocus` option (matching `data-disable-auto-focus`)
+- the `NotificationBanner` component, for its `disableAutoFocus` option (matching `data-disable-auto-focus`)
+
+You can omit these configuration options when calling the Nunjucks macro and provide them:
+
+- at component instantiation, in a configuration object as second argument
+- when initialising components in bulk using `initAll`
```

Comment:
```suggestion
- initialising components in bulk using `initAll`
```

Reply:
added some minor suggestions for clarity. 
Just checking whether there are any acknowledgements to contributors to add?

Reply:
@claireashworth I'm not sure what you mean by "acknowledgements to contributors to add". Do you mean "
acknowledging external contributors? If so, we're good for that entry as it was Ollie's and mine doing 😄

Reply:
👍🏻 That's exactly what I meant. Just checking for when we make release notes as well.